presentation-di/src/main/java/com/wednesday/template/presentation/Presentation.kt

```diff
@@ -0,0 +1,11 @@
+package com.wednesday.template.presentation
+
+import com.wednesday.template.presentation.weather.search.SearchFragmentViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val searchViewModel = module {
```

Comment:
Let's name this viewModelModule
```suggestion
val viewModelModule = module {
```

Reply:
Ok